packages/console/Command/ExportListCommandsCommand.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Draw\Component\Console\Command;
+
+use Symfony\Component\Process\Process;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand(name: 'app:list-commands')]
+class ExportListCommandsCommand extends Command
+{
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('List all available Symfony commands with their documentation.')
+            ->addOption('output', 'o', InputOption::VALUE_REQUIRED, 'Output file', 'commands_help.txt');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $application = $this->getApplication();
+        $outputFile = $input->getOption('output');
+
+        foreach ($application->all() as $command) {
+            $process = new Process(['bin/console', $command->getName(), '--help']);
+            $process->mustRun();
+
+            if ($process->isSuccessful()) {
+                file_put_contents($outputFile, $this->addNameToTheFullDescription($this->generateBlockText($command->getName()), $process->getOutput()), FILE_APPEND);
```

Comment:
you are missing some line break between each command. If you look at your output it's a bit sqeeze.